test/unit/routes/signed-out-routes.test.js

```diff
@@ -0,0 +1,39 @@
+import { vi, beforeEach, describe, test, expect } from 'vitest'
+import { signedOut } from '../../../src/routes/signed-out-routes'
```

Comment:
```suggestion
// Thing under test
import { signedOut } from '../../../src/routes/signed-out-routes'
```

Reply:
We want to keep code comment minimal, so I don't consider the comment line to be necessary.